src/omotes_sdk/internal/worker/worker.py

```diff
@@ -263,21 +272,30 @@ def wrapped_worker_task(
     logger.info("Worker started new task %s with reference %s", job_id, job_reference)
     task_util = TaskUtil(job_id, task, task.broker_if)
     task_util.send_start()
-    output_esdl = WORKER_TASK_FUNCTION(input_esdl, params_dict, task_util.update_progress)
-
-    input_esh = pyesdl_from_string(input_esdl)
-    input_energy_system: EnergySystem = input_esh.energy_system
-    if job_reference is None:
-        new_name = f"{input_energy_system.name}_{WORKER_TASK_TYPE}"
-    elif job_reference == "":
-        new_name = f"{input_energy_system.name}"
+    output_esdl, esdl_messages = WORKER_TASK_FUNCTION(
+        input_esdl, params_dict, task_util.update_progress
+    )
+
+    logger.warning(f"WORKER_TASK_FUNCTION output: {esdl_messages}, \n{output_esdl}, {task}")
```

Comment:
Logging is usually done in the format of string interpolation:
```suggestion
    logger.warning(f"WORKER_TASK_FUNCTION output: %s, \n%s, %s", esdl_messages, output_esdl, task)
```

The reason for this is performance. Currently the string will always be generated even if the loglevel is set to ERROR. If you use the above notation, the string will only be created if the logger actually forwards the log message. For a single logmessage this is neglible but using this format instead of f-strings will add up over time.

Reply:
this was actually remnant testing output, thanks for catching :-)